tests/unit/test_backend_dist.py

```diff
@@ -126,6 +126,19 @@ def test_npartitions_with_num_executors(self):
 
         self.assertEqual(backend.npartitions, 10)
 
+    def test_npartitions_with_already_existing_spark_context(self):
+        """
+        Test case to check that the value of `npartitions` is correctly set
+        when a Spark Context already exists.
+
+        """
+        from pyspark import SparkContext, SparkConf
+        sparkConf = SparkConf().set('spark.executor.instances', 15)
+        sc = SparkContext.getOrCreate(sparkConf)
```

Comment:
I would do here `sc = SparkContext(sparkConf)` instead of `getOrCreate`, in this test you want to make sure you create it with the configuration you are passing.

Reply:
👍